quality/metrics/bertscore.md

```diff
@@ -0,0 +1,19 @@
+---
+grand_parent: Building and research
+parent: Metrics
+nav_order: 7
+title: BERTScore
+description:
+---
+
+The **BERTScore** metric calculates the similarity in meaning between the machine translation output and a reference.
+BERTScore uses contextual embeddings.
```

Comment:
I'd remove this.  People who know how BERT works don't need it, people who don't know how BERT works won't know after reading this.

The place to explain a bit would be in or after this sentence further down:
> The BERTScore metric uses BERT sentence representation.